static/js/dropdown_list_widget.js

```diff
@@ -129,15 +129,17 @@ export const DropdownListWidget = function ({
         });
 
         search_input.on("keydown", (e) => {
-            if (!/(38|40|27)/.test(e.keyCode)) {
+            const {key, keyCode, which} = e;
+            if (!/(ArrowUp|ArrowDown|Escape)/.test(key)) {
```

Comment:
Can we move to a non-regexy way to check whether it's one of those 3 values?  A `switch` statement is one option.